test/containers/SuperpowerIndexContainer.js

```diff
@@ -0,0 +1,27 @@
+import SuperpowerIndexContainer from '../../app/javascript/react/containers/SuperpowerIndexContainer';
+import SuperpowerTile from '../../app/javascript/react/components/SuperpowerTile';
+
+describe('SuperpowerIndexContainer', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <SuperpowerIndexContainer
+      />
+    )
+  });
+
+  it('should have the specified initial state', () => {
+    expect(wrapper.state()).toEqual({ superpowers: [] });
+  })
+
+  it('should render a SuperpowerTile Component', () => {
+    expect(wrapper.find(SuperpowerTile)).toBePresent();
+  });
+
+
+  it('renders a div with a list of superpowers', () => {
+    expect(wrapper.contains(<div>'Flight'</div>)).toEqual(true);
```

Comment:
Likewise, I'm pretty sure this test isn't running: we didn't put the superpower with a name of "Flight" into state, so I don't see how it could be rendering to the page!